src/core/oplist

```diff
@@ -927,3 +927,5 @@ prof_allocated   .s r(obj)
 ctw_check        .s r(obj) int16
 
 coverage_log     .s str int32 int32 int64
+
+hardware_concurrency w(int64) :pure
```

Comment:
Please read the instructions at the top of `oplist` to see how to correctly place new ops in this file. Also, the name should not contain a `_`, we only use that for type suffixes and dynamic op prefixes. I'd prefer a name like `cpucores` or so, which makes clearer what it is.